### PR TITLE
Hostname parsing fix

### PR DESCRIPTION
**Pull request by mkulaczkowski**

### CODE REVIEW

File: gradient_utils/metrics.py [hunk 1]
```diff
@@ -21,20 +21,26 @@ def _get_env_var_or_raise(*env_vars):
...
     if os.getenv('PAPERSPACE_EXPERIMENT_ID'):
         return os.getenv('PAPERSPACE_EXPERIMENT_ID')
     hostname = os.getenv('HOSTNAME', "")
     try:
-        experiment_id = hostname.split('-')[1]
-        return experiment_id
+        object_id = hostname.split('-')
+
+        if len(object_id) == 1:  #for Noteboooks
+            return object_id
```
Comment (BartoszCki) [inline gradient_utils/metrics.py:32]: This one would be a one element list not a string

File: gradient_utils/metrics.py [hunk 2]
```diff
@@ -21,20 +21,26 @@ def _get_env_var_or_raise(*env_vars):
...
+def _get_object_id():
     if os.getenv('PAPERSPACE_EXPERIMENT_ID'):
         return os.getenv('PAPERSPACE_EXPERIMENT_ID')
     hostname = os.getenv('HOSTNAME', "")
     try:
-        experiment_id = hostname.split('-')[1]
-        return experiment_id
+        object_id = hostname.split('-')
+
+        if len(object_id) == 1:  #for Noteboooks
```
Comment (kossak) [inline gradient_utils/metrics.py:31]: nit: pycharm might have objections against lack of space after #

Comment (mkulaczkowski) [reply gradient_utils/metrics.py:31]: Fixing

File: gradient_utils/metrics.py (hunk 1 above)
Comment (mkulaczkowski) [reply gradient_utils/metrics.py:32]: Fixing

**Pull request merged**